pkg/tf2pulumi/convert/eject_test.go

```diff
@@ -74,6 +78,50 @@ func isTruthy(s string) bool {
 	return s == "1" || strings.EqualFold(s, "true")
 }
 
+func formatTestFile(src io.Reader, dst io.Writer, isExperimental bool) error {
```

Comment:
Can we change the name of the function to `expandPragmas` or something similar. I would not call a pre-processor "formatting". I also think it needs a comment explaining what its doing, and the intended grammar.